py-polars/polars/dataframe/frame.py

```diff
@@ -5066,6 +5066,29 @@ def limit(self, n: int = 5) -> Self:
         See Also
         --------
         head
+
+        Examples
+        --------
+        Get the first 3 rows of a DataFrame.
+        >>> df = pl.DataFrame(
```

Comment:
This needs a blank line or it won't render correctly:

```suggestion
        Get the first 3 rows of a DataFrame.
        
        >>> df = pl.DataFrame(
```